src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -2812,6 +2813,66 @@ def test_sql_managed_instance_mgmt(self):
                  expect_failure=True)
 
 
+class SqlManagedInstancePoolScenarioTest(ScenarioTest):
+    @record_only()
+    def test_sql_instance_pool(self):
+
+        instance_pool_name = self.create_random_name(instance_pool_name_prefix, managed_instance_name_max_length)
+        subnet = "/subscriptions/a8c9a924-06c0-4bde-9788-e7b1370969e1/resourceGroups/billingPools/providers/Microsoft.Network/virtualNetworks/vnet-billingpool1/subnets/InstancePool"
+        license_type = 'LicenseIncluded'
+        location = 'northcentralus'
+        v_cores = 8
+        edition = 'GeneralPurpose'
+        family = 'Gen5'
+        resource_group = "billingPools"
+
+        # test create sql managed_instance
+        self.cmd(
+            'sql instance-pool create -g {} -n {} -l {} '
+            '--subnet-id {} --license-type {} --capacity {} -e {} -f {}' .format(
```

Comment:
It looks --subnet-id is not in your command. What do you have is --subnet according to https://github.com/Azure/azure-cli/pull/11721/files#diff-d300fa90ed2ac1c25680915bc5febaf2R1000.

Reply:
Fixed this in my next iteration